### DISCUSSION TITLE
어떻게 된겁니까 ㅜㅜ

### DISCUSSION THREAD

**Discussion by Swipe0801** [devrel-kr/HackaLearn#425] [9 잡담방]
5일간의 모든 열정이 다 날라간거 같아서 지금 팀원분들이 다들 해탈하시고 있습니다..정말 공들여서 만든 작품인데 이렇게 날라간다면 정말 어떻게 되는겁니까..ㅜ

**Comment by justinyoo** [comment]
@Swipe0801 8월 16일 0시 전에 생성한 PR에 대해서는 12시간의 PR 수정 기간을 드렸구요, 8월 16일 12시를 넘어서까지 수정 요청이 반영되지 않은 PR들은 모두 자동으로 닫혔습니다. 따라서 해당 부분에 대해서는 안타깝지만 반영되지 않습니다.
